st2common/tests/unit/test_db_execution.py

```diff
@@ -188,19 +191,6 @@ def test_execution_inquiry_secrets(self):
             "supersecretvalue",
         )
 
-    def test_execution_inquiry_response_action(self):
```

Comment:
todo: why did I remove this test?

Reply:
the live action is no longer embedded in the execution returned here so there is nothing to mask.

Reply:
I suppose we can move this test over to the live action model to be sure.